### ISSUE TITLE
Same pre-trained weights for MNIST

### ISSUE THREAD

**Issue by nurlanov-zh** [eth-sri/SABR#1]
Hi, thanks for sharing your code and pre-trained weights. 

The links to weights for MNIST 0.1 and MNIST 0.3 are the same. Is it correct?

```
Please use the following links to download our trained models:

[Mnist 0.1](https://files.sri.inf.ethz.ch/sabr/SABR_mnist_01_best.pynet)  
[Mnist 0.3](https://files.sri.inf.ethz.ch/sabr/SABR_mnist_01_best.pynet)  
```

**Comment by nurlanov-zh** [comment]
Another question is that the architectures in pre-trained weights do not correspond to default configuration. E.g. in MNIST and CIFAR-10, the pretrained weights do not have values for batch norm after linear layer. 

What is the architecture for weights in `[TinyImageNet 1/255](https://files.sri.inf.ethz.ch/sabr/SABR_TIN_1_best.pynet)`?

**Comment by mnmueller** [comment]
Hi @nurlanov-zh,

Thanks for your interest. Indeed there was an error in the link for the MNIST 0.3 weights that is fixed now.

Regarding the architecture: We trained the networks with a batch norm after the linear layer, but merged its weights with that linear layer for analysis with MN-BaB as that does not support 1D BatchNorm.

For TinyImageNet, we also use a CNN7 architecture. To load the net, feel free to reference the loading code in MN-BaB (https://github.com/eth-sri/mn-bab/blob/SABR_ready/src/utilities/loading/network.py#L798) or refer to the corresponding code in this repo (https://github.com/eth-sri/SABR/blob/main/src/networks.py#L126).

 I hope this addresses all your questions.
 
 Cheers,
 Mark
